testing.md

```diff
@@ -1,4 +1,4 @@
-# Testing
+git@github.com:lasselehtinen/docs.git# Testing
```

Comment:
I don't know why the repo was pasted on the beginning of the file, this has been now fixed.